charts/calico/templates/calico-node-rbac.yaml

```diff
@@ -138,26 +123,6 @@ rules:
       - create
       - update
 {{- if eq .Values.network "calico" }}
-  # These permissions are required for Calico CNI to perform IPAM allocations.
```

Comment:
I think with testing you'll find that calico/node also needs some of these permissions.